testsuite/features/core_srv_menu.feature

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2017-2019 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Web UI menu
```

Comment:
1. I think this should go to secondary features - it's not a preparation
2. We already have a feature that does check menus - please merge both features (as secondary)

Reply:
>     1. I think this should go to secondary features - it's not a preparation

I had the same thinking, but then: why not to test it at the very begin? if the menu is broken, the most of the tests will fail as well. I though about this: I start the testsuite run, I see the menu is broken, I quickly abort the running and fix it soon. If you think this is not that much relevant, I will just move it to a secondary feature, as I agree about it's not a preparation.

Reply:
For now fine to keep at core section yep.
But when we decide to use the testsuite in our PRs, then the developer will want to run the tests more important for their changes excluding whatever it takes unnecessary extra time. We are not at that point yet, so fine to let it at core.

Reply:
Nice POV, I didn't thought about it. Great, I just squashed all menu scenarios into one in https://github.com/uyuni-project/uyuni/pull/937/commits/dad89c163b875c03668ca18b273e96321ad88792 . We will move it down in the order once testsuite on PRs will be in place :+1:

Reply:
I suggest a compromise. Why not move it very top of secondary ?

That way, if a developer just wants a setup, he/she will not lose the time testing the menus. But it would still be tested very early.

Is that acceptable?